_field-types/mapping-parameters/index-phrases.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Index phrases
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 80
+has_children: false
+has_toc: false
+---
+
+# Index phrases
+
+The `index_phrases` mapping parameter determines whether a field’s text is additionally processed to generate phrase tokens. When enabled, the system creates extra tokens representing sequences of exactly two consecutive words (`bigrams`). This can significantly improve the performance and accuracy of phrase queries. However, it also increases the index size and the time needed to index documents.
```

Comment:
```suggestion
The `index_phrases` mapping parameter determines whether a field's text is additionally processed to generate phrase tokens. When enabled, the system creates extra tokens representing sequences of exactly two consecutive words (_bigrams_). This can significantly improve the performance and accuracy of phrase queries. However, it also increases the index size and the time needed to index documents.
```